.github/workflows/tag.yml

```diff
@@ -0,0 +1,22 @@
+name: Tag
+on: workflow_dispatch
+
+jobs:
+  tag-job:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout current code
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.BROADBOT_TOKEN }} # this allows the push to succeed later
+      - name: Bump the tag to a new version
+        # https://github.com/DataBiosphere/github-actions/tree/master/actions/bumper
+        uses: databiosphere/github-actions/actions/bumper@bumper-0.0.4
+        id: tag
+        env:
+          GITHUB_TOKEN: ${{ secrets.BROADBOT_TOKEN }}
+          HOTFIX_BRANCHES: hotfix.*
```

Comment:
Are we still doing `hotfix` branches in the new continuous deployment world?

I would love to favor continuous + seamless deployment early on, and make accommodations for slower releases later if necessary. Because it's really hard to go in the other direction (making things more continuous).

Reply:
The terra service hotfix process is documented [here](https://github.com/broadinstitute/terra-helmfile/blob/master/docs/HOTFIX.md). I think the hotfix process is necessary until we break apart the monolith release process.

Reply:
Great question. This looks optional in the action so I'll happily remove it.

Reply:
@pshapiro4broad because Cromwell is an app I think our deployments will necessarily be separate from the monolith from the start anyway

Reply:
@pshapiro4broad you are right about the monolith of course, but as Chris says this service is intended as "never was in the monolith" versus "needs to break out of the monolith".

Reply:
Oh, I didn't know that about cbas. That's good!

Reply:
We should make sure to consult with Ambica & QA about these things. We still need to run through the new service checklist and if there are differences due to the app model, then we should document them there rather than us accidentally going rogue.

Reply:
The recommendation is to review [the checklist](https://docs.google.com/document/d/1deV-i3_oMGBwUreDUKEhovdLawBdf0feshcP1h11wR0/edit#heading=h.sogi2y5vnhy) at the start of development, fill in parts as possible during development (now), and have a review meeting before designating the service as in-production (long way off).